macOS/LocalPackages/NewTabPage/Tests/NewTabPageTests/NewTabPageConfigurationClientTests.swift

```diff
@@ -35,6 +36,7 @@ final class NewTabPageConfigurationClientTests: XCTestCase {
         contextMenuPresenter = CapturingNewTabPageContextMenuPresenter()
         eventMapper = CapturingNewTabPageConfigurationEventHandler()
         client = NewTabPageConfigurationClient(
+            sectionsAvailabilityProvider: MockNewTabPageSectionsAvailabilityProvider(),
```

Comment:
We could add tests to cover omnibar not being available, by adding a variant of `testThatInitialSetupReturnsConfiguration`.